README.md

```diff
@@ -37,3 +37,24 @@ npm install -g webpack
 npm install
 ```
 
+## API routes
+
+## restaurant-specific routes
+GET /restaurant:id - loads restaurant data using unique restaurant name as parameter
```

Comment:
You mention name as the parameter, but here the parameter is id. You might want to change the param to match your comment or vice versa.